WolvenKit.Common/Model/Arguments/ImportArgs.cs

```diff
@@ -101,6 +101,17 @@ public XbmImportArgs()
     /// </summary>
     public class GltfImportArgs : ImportArgs
     {
+        /// <summary>
+        /// Should a Material.Json be imported?
+        /// </summary>
+        [Category("Import Settings")]
+        [Display(Name = "Import with Material.Json")]
+        [Description("If selected materials will be updated from a Material.json file.")]
+        public bool importMaterials { get; set; } = false;
```

Comment:
small nitpick, but can this be UpperCase?

Reply:
the Json? 😅

Reply:
the property name :D `importMaterials `

Reply:
Oh, yeah. Sure. OMW.

Reply:
Done. Also renamed some of the other properties that were lower-case.